code-challenges.test.js

```diff
@@ -25,27 +42,85 @@ const colors2 = [
   "aquamarine",
   "saffron"
 ]
+
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
+// pseudo code
+// input: a function that take in an array
+// output: a subset array with the first item removed and shuffled content
+// used the shift method to remove the first item in the array and the sort to shuffle the elements then .random
```

Comment:
This does not adequately explain the logic happening in your code, focus on elaborating more and understanding what you're using